routes/password_check.js

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
```

Comment:
LGTM, thanks @shane-tomlinson.  I wonder if we should put these new route utils under a sub-module like e.g. routes/utils/ to make it clear that they don't contain any actual routing code.  That's a very minor housekeeping preference though.